codemeta.json

```diff
@@ -1,616 +1,809 @@
-                "name": "Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St., 02138 Cambridge MA, US"
-            },
-            "familyName": "Donath",
-            "givenName": "Axel"
+    "codeRepository": "https://github.com/gammapy/gammapy",
+    "datePublished": "2014-08-25",
+    "description": "Gammapy analyzes gamma-ray data and creates sky images, spectra and lightcurves, from event lists and instrument response information; it can also determine the position, morphology and spectra of gamma-ray sources. It is used to analyze data from H.E.S.S., Fermi-LAT, HAWC, and the Cherenkov Telescope Array (CTA).",
+    "identifier": "https://doi.org/10.5281/zenodo.4701488",
+    "keywords": [
+        "Astronomy",
+        "Gamma-rays",
+        "Data analysis"
+    ],
+    "license": "https://spdx.org/licenses/BSD-3-Clause",
+    "name": "Gammapy: Python toolbox for gamma-ray astronomy",
+    "url": "https://gammapy.org/",
+    "version": "1.0",
+    "Organization": "The Gammapy project",
+    "applicationCategory": "Astronomy",
+    "applicationSubCategory": "Very-high energy gamma rays",
+    "programmingLanguage": "Python 3",
+    "operatingSystem": "Linux, Mac OS, Windows",
+    "audience": {
+        "@id": "/audience/science-research",
+        "@type": "Audience",
+        "audienceType": "Science/Research"
+    },
+    "maintainer": {
+        "@id": "https://orcid.org/0000-0003-4568-7005",
+        "@type": "Person",
+        "affiliation": {
+            "@type": "Organization",
+            "name": "Center for Astrophysics | Harvard & Smithsonian, CfA, 60 Garden St., 02138 Cambridge MA, US"
         },
+        "familyName": "Donath",
+        "givenName": "Axel"
+    },
+    "developmentStatus": "active",
+    "issueTracker": "https://github.com/gammapy/gammapy/issues",
+    "funder": "None, apart for the permanent staff",
+    "dateCreated": "2014-01-01",
+    "referencePublication": {
+        "authors": [
+            {
+                "family-names": "Deil",
+                "given-names": "Christoph"
+            },
+            {
+                "family-names": "et al."
+            }
+        ],
+        "title": "Gammapy - A prototype for the CTA science tools",
+        "doi": "10.48550/ARXIV.1709.01751",
+        "url": "https://arxiv.org/abs/1709.01751",
+        "type": "conference-paper",
+        "year": 2017,
+        "conference": {
+            "name": "Proc. 35th ICRC, Busan, South Korea, PoS(ICRC2017)766"
+        },
+        "publisher": {
+            "name": "arXiv"
+        }
+    },
+    "dateModified": "2022-09-15",
+    "email": "GAMMAPY-COORDINATION-L@IN2P3.FR",
+    "readme": "If you use this software, please cite it using the metadata from this file.",
+    "runtimePlatform": [
+        "C",
+        "Cython",
+        "Python 3",
+        "Python 3.8",
+        "Python 3.9",
+        "Python 3.10",
+        "Python Implementation CPython"
+    ],
+    "softwareRequirements": [
```

Comment:
The software requirements listed here correspond to the dev environment. Many are not needed.
Also should we put minimum requirements as per setup.cfg or the maximal list that will go in the `environment-v1.0.yaml`?

Reply:
I think this entry should be auto generated from the `setup.cfg` and just list the minimal requirements. The information in the `setup.cfg` is much more detailed. We shouldn't even try to duplicate it here min any way...